app/src/main/res/layout/activity_main.xml

```diff
@@ -59,13 +60,13 @@
 
             <TextView
                 style="@style/WhiteText"
-                android:text="Contact" />
+                android:text="Password" />
 
 
             <EditText
                 style="@style/editText"
                 android:ems="10"
-                android:inputType="textPersonName"
+                android:inputType="textPassword"
```

Comment:
@lilivoica, what did you do here? What did you intend?

Reply:
I intended to correct the inputType to password.

Reply:
Aha, ok, thanks for clarifications!